HISTORY.md

```diff
@@ -1,4 +1,4 @@
-## 2017-04-24 - Release 2.4.1 ([diff](https://github.com/locp/cassandra/compare/2.4.0...2.4.1))
+## [2.4.1](https://github.com/voxpupuli/puppet-cassandra/compare/2.4.0...2.4.1) (2017-04-24)
```

Comment:
The *exact* format of the first release in the HISTORY.md has to be correct for gcg to work correctly.  (even a trailing space breaks it!)